docker-build

```diff
@@ -1,2 +1,16 @@
 #!/bin/bash
+set -e
+if [ -z "$(type -f docker 2>/dev/null)" ]; then
```

Comment:
Instead of duplicating the docker command check here and in `docker-run` can you put it in a separate `util` script like the Ansible command check?